app/controllers/comments_controller.rb

```diff
@@ -2,34 +2,39 @@
 
 class CommentsController < ApplicationController
   before_action :set_comment, only: [:update, :destroy]
+  before_action :set_bill, only: :create
 
   def create
-    @comment = Comment.new(comment_params)
+    @comment = @bill.comments.new(comment_params)
     if @comment.save
-      redirect_to bill_path(@comment.bill), notice: t("comment.create.success")
+      redirect_to bill_path(@comment.bill), notice: t(".success")
     else
       render bill_path(@comment.bill)
     end
   end
 
   def update
     if @comment.update(comment_params)
-      redirect_to bill_path(@comment.bill), notice: t("comment.update.success")
+      redirect_to bill_path(@comment.bill), notice: t(".success")
     else
       render bill_path(@comment.bill)
     end
   end
 
   def destroy
     @comment.destroy
-    redirect_to bill_path(@comment.bill), notice: t("comment.destroy.success")
+    redirect_to bill_path(@comment.bill), notice: t(".success")
   end
 
   private
     def set_comment
       @comment = Comment.find(params[:id])
     end
 
+    def set_bill
+      @bill = Bill.find(params[:bill_id])
+    end
+
     def comment_params
       params.require(:comment).permit(:user_id, :bill_id, :description)
```

Comment:
[MUST] ここも user_id 消しておきましょ〜👍

```suggestion
      params.require(:comment).permit(:bill_id, :description)
```

他にも↓で user_id が残ってますね😁

https://github.com/chihaso/bill_watcher/blob/8e7edde5ecd8fe3491df1a60a0e0662368dfda10/test/fixtures/comments.yml#L4
https://github.com/chihaso/bill_watcher/blob/8e7edde5ecd8fe3491df1a60a0e0662368dfda10/test/fixtures/comments.yml#L9
https://github.com/chihaso/bill_watcher/blob/8e7edde5ecd8fe3491df1a60a0e0662368dfda10/db/migrate/20200423004432_create_comments.rb#L6
https://github.com/chihaso/bill_watcher/blob/8e7edde5ecd8fe3491df1a60a0e0662368dfda10/db/schema.rb#L30

マイグレーションファイルと schema.rb に関しては、今回の場合は新しくマイグレーションファイルを追加するのではなくdb:rollback でバージョンを戻した後に、直接マイグレーションファイルを編集して再度 db:migrate を実行する形でいいと思います〜😄

Reply:
@sanfrecce-osaka ありがとうございます！！
確認させてしまってすみません😭
修正いたしました！
https://github.com/chihaso/bill_watcher/commit/790cebef5a926cad7eb00cd24ca8a622b531912b